index.html

```diff
@@ -219,7 +230,7 @@ <h3>HTML</h3>
                                 <h3>CSS</h3>
                             </li>
                         </ul>
-                    <img  class="separator"  src="css/img/Separator-Bottom.png">
+                    <img id="separator1" class="separator"  src="css/img/Separator-Bottom.png">

```

Comment:
- [ ] please, while you did a great job in the desktop design, remember the mobile version still has to look good. kindly fix the contact part of the mobile version, so your design can be on point. 

![Screenshot (1142)](https://github.com/BiancaMe/Portfolio-Mobile/assets/65298048/f9e5a55b-8830-40e2-9fc8-c2c8db3149de)